config/resource-manager.hcl

```diff
@@ -374,6 +374,7 @@ service "orbital" {
 service "paloaltonetworks" {
   name      = "PaloAltoNetworks"
   available = ["2022-08-29"]
+  ignore    = ["2022-08-29"]
 }
```

Comment:
I don't _think_ this works, we'd either need to remove the block for now - or fix the casing and move forward?

Reply:
Removed.